Source/engine/render/automap_render.cpp

```diff
@@ -22,13 +23,22 @@ template <DirectionX DirX, DirectionY DirY>
 void DrawMapLine(const CelOutputBuffer &out, Point from, int height, std::uint8_t colorIndex)
 {
 	while (height-- > 0) {
-		out.SetPixel(from, colorIndex);
-		from.x += static_cast<int>(DirX);
-		out.SetPixel(from, colorIndex);
-		from.x += static_cast<int>(DirX);
-		from.y += static_cast<int>(DirY);
+		if(sgOptions.Gameplay.bDrawAutomapBorder == true) {
+			out.SetPixel({ from.x, from.y + 1 }, 0);
+		}
+			out.SetPixel(from, colorIndex);
+			from.x += static_cast<int>(DirX);
+		if(sgOptions.Gameplay.bDrawAutomapBorder == true) {
+			out.SetPixel({ from.x, from.y + 1 }, 0);
+		}
+			out.SetPixel(from, colorIndex);
+			from.x += static_cast<int>(DirX);
+			from.y += static_cast<int>(DirY);
 	}
-	out.SetPixel(from, colorIndex);
+	if(sgOptions.Gameplay.bDrawAutomapBorder == true) {
+		out.SetPixel({ from.x, from.y + 1 }, 0);
+	}
+		out.SetPixel(from, colorIndex);

```

Comment:
```suggestion
	out.SetPixel(from, colorIndex);
```

Reply:
I shouldn't have moved the lines any further to the right side.